catalogs/management/commands/load_catalog.py

```diff
@@ -41,7 +41,7 @@ def _load_catalog(self, input_file: Path, name: str, **catalog_args):
 
         try:
             with open(input_file, mode="rb") as catalog:
-                file = File(catalog)
+                file = File(catalog, name=input_file)
```

Comment:
This looks like you fixed a bug!

Reply:
It's... something. Technically the thing I was trying to fix is still kind of there, but just got booted. Django does some stuff to "media" fields that results in weird behavior that I don't think that we really need in our case since we're effectively loading a "static" dataset from the repo. I was torn between trying to make it work vs updating that field to _not_ be a "media" field. Wasn't sure I could justify this one as "in-scope". 

Effectively though, I don't think it'll make too much of a difference in the deployed environments, but for local development, if you delete your local "media" folder every now and again you may get a `FileNotFound` error because of the magic django tries to do in the background to protect the "original" files (effectively makes copies and references the copies in certain circumstances).